libraries/community/p2/All/BinFloat/BinFloat.spin2

```diff
@@ -1028,6 +1028,7 @@ PUB Unpack10(x) : sign, exp10, decval | loptr, hiptr, midptr, binval, binexp
   SendFloatSci: Stream out scientific representation of a binary float x
 }}
 PUB SendFloatSci(x) | s, exp, decval, digit, nopoint
+  nopoint := 1
```

Comment:
I urge you to break this out into a separate commit, since the rest of this commit is whitespace related.

Reply:
Whitespace is significant in the Spin language, so the whitespace commit isn't just cosmetic, it fixes a bug too. The whole thing is required.

Reply:
Then why is the subject of this PR:

  "fixed some spacing issues and an uninitialized variable in BinFloat.spin2"

I understand that both issues need to be fixed atomically, but even the name of this PR indicates that TWO issues are being fixed here, whitespace and the uninitialized variable.

Not suggesting two PRs, just two commits in this PR.

In any case, it's your PR, I was just trying to be helpful.